addons/web/static/src/js/chrome.js

```diff
@@ -706,9 +706,9 @@ instance.web.client_actions.add("reload_context", "instance.web.ReloadContext");
  * If can't go back in history stack, will go back to home.
  */
 instance.web.HistoryBack = function(parent) {
-    if (!parent.history_back()) {
+    $.when(parent.history_back()).fail(function() {
```

Comment:
no need $.when here because history_back method return deffer object in any condition.